### PR TITLE
Updated User API's parameter name in openapi.yml

### PR DESCRIPTION
**Pull request by aashir21**
# Why?

To maintain uniformity between parameter name.

- Related to #612

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml [hunk 1]
```diff
@@ -298,9 +298,9 @@ paths:
       - Users API
       security: []
       parameters:
-        - name: name
+        - name: loginId
```
[the remark this answers is not in the document]
Comment (techcobweb) [reply galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml:301]: > summary: Returns user details by extracting claims from token
Should be "Query user details"

File: galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml [hunk 2]
```diff
@@ -298,9 +298,9 @@ paths:
...
           in: query
-          description: The name of the client, extracted from the token. #Me is a special keyword reserved to return details about currently logged in user.
+          description: The loginId of the client, extracted from the token. #Me is a special keyword reserved to return details about currently logged in user.
```
Comment (techcobweb) [inline galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml:303]: > The loginId of the client, extracted from the token.
The loginId of the user whose details will be returned.

**Review by techcobweb — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**